### PR TITLE
Add cheat-sheet for developers

### PR DESCRIPTION
**Pull request by mitelg**

### CODE REVIEW

File: source/developers-guide/cheat-sheet/index.md [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+public static function getSubscribedEvents()
+{
+    return [
+        'CLASS::FUNCTION::TYPE' => 'LISTENER',
+    ];
+}
+```
+
+## Attributes
+### Creating a new attribute
```
Comment (t2oh4e) [inline source/developers-guide/cheat-sheet/index.md]: I think you should add a hint that this can also be used to update existing attributes :)

Comment (mitelg) [inline source/developers-guide/cheat-sheet/index.md]: done :+1:

File: source/developers-guide/cheat-sheet/index.md [hunk 2]
```diff
@@ -0,0 +1,233 @@
...
+</div>
+
+## Templating
+Have a look here for a <a href="http://community.shopware.com/files/downloads/templatecheatsheeten-12249471.pdf" target="_blank">Smarty Cheat-Sheet</a>
+
+## Events and hooks
+Use your main plugin class or a subscriber class which implements the `\Enlight\Event\SubscriberInterface` to register new events or hooks
+
+### Events
+```php
```
Comment (janbuecker) [inline source/developers-guide/cheat-sheet/index.md]: Please describe shortly, how the position affects the execution order. (higher number vs lower number and the default)

Comment (mitelg) [inline source/developers-guide/cheat-sheet/index.md]: done

**Pull request merged**